frontend/src/Settings/Indexers/Options/IndexerOptions.js

```diff
@@ -97,7 +97,7 @@ function IndexerOptions(props) {
               <FormInputGroup
                 type={inputTypes.NUMBER}
                 name="availabilityDelay"
-                unit="Days"
+                unit="days"
```

Comment:
@ta264 do you think this and some other units in the frontend could become a translate?

Reply:
Yes